src/helpers.php

```diff
@@ -13,6 +13,14 @@ function text(string $label, string $placeholder = '', string $default = '', boo
     return (new TextPrompt(...func_get_args()))->prompt();
 }
 
+/**
+ * Prompt the user for multiline text input.
+ */
+function textarea(string $label, int $rows = 5, string $placeholder = '', string $default = '', bool|string $required = false, ?Closure $validate = null, string $hint = ''): string
```

Comment:
Wondering if the `$rows` param should be moved to after the `$default` param (the same position as the similar `$scroll` parameter on `select` and `multiselect`)